app/src/features/resources/ResourcesLayout.tsx

```diff
@@ -0,0 +1,35 @@
+import Box from '@material-ui/core/Box';
```

Comment:
not an issue or question pertaining specifically to this pr, but just curious about if we can just have a single layout component cause all the layout components (resources, permits, projects, etc) are all the same code pretty much right?

Reply:
yeah .. at the moment, we don't have a layout for the resources yet.  But good idea.